ingestion/src/metadata/ingestion/source/dashboard/domodashboard/metadata.py

```diff
@@ -151,25 +151,25 @@ def yield_dashboard_chart(
                 )
 
                 if filter_by_chart(
-                    self.source_config.chartFilterPattern, chart["title"]
+                    self.source_config.chartFilterPattern, chart.get("title")
                 ):
-                    self.status.filter(chart["title"], "Chart Pattern not allowed")
+                    self.status.filter(chart.get("title"), "Chart Pattern not allowed")
                     continue
 
                 yield CreateChartRequest(
                     name=chart_id,
                     description=chart.get("description", ""),
                     displayName=chart.get("title"),
                     chartType=get_standard_chart_type(
-                        chart["metadata"].get("chartType", "")
+                        chart.get("metadata").get("chartType", "")
                     ).value,
                     chartUrl=chart_url,
                     service=EntityReference(
                         id=self.context.dashboard_service.id.__root__,
                         type="dashboardService",
                     ),
                 )
-                self.status.scanned(chart["title"])
+                self.status.scanned(chart.get("title"))
             except Exception as exc:
                 logger.warning(f"Error creating chart [{chart}]: {exc}")
                 logger.debug(traceback.format_exc())
```

Comment:
Should we log an error in the status in case it fails?

Reply:
yes 👍